New release?

There's a fix for OpenBSD in cddb67b8534a46a372beb9fd63c5ce8d1473cbda, but this has not been released. Without this some crates depending on `socket2` are not buildable in OpenBSD.

```
error[E0425]: cannot find value `KEEPALIVE_TIME` in this scope
   --> /home/oherrala/.cargo/registry/src/github.com-1ecc6299db9ec823/socket2-0.4.1/src/sys/unix.rs:871:46
    |
871 |         getsockopt::<c_int>(fd, IPPROTO_TCP, KEEPALIVE_TIME)
    |                                              ^^^^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,14 @@
+# 0.4.2
+
+## Added
+
+* `Socket::(set_)freebind_ipv6`, getter and setter for `IPV6_FREEBIND`.
+
+## Fixed
+
+* Compilation on OpenBSD.
+* Usage of incorrect flag in `Socket::(set_)freebind`.
+
 # 0.4.1
 
 ## Added

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name          = "socket2"
-version       = "0.4.1"
+version       = "0.4.2"
 authors       = [
   "Alex Crichton <alex@alexcrichton.com>",
   "Thomas de Zeeuw <thomasdezeeuw@gmail.com>"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -323,7 +323,9 @@ impl<'a> DerefMut for MaybeUninitSlice<'a> {
 #[derive(Debug, Clone)]
 pub struct TcpKeepalive {
     time: Option<Duration>,
+    #[cfg_attr(target_os = "redox", allow(dead_code))]
     interval: Option<Duration>,
+    #[cfg_attr(target_os = "redox", allow(dead_code))]
     retries: Option<u32>,
 }
 

